CONTRIBUTING.md

```diff
@@ -101,10 +101,10 @@ Run the build script
 $ npm run build
 ```
 
-Run the Grunt tools:
+Build a local copy of jQuery using grunt
 
 ```bash
-$ grunt && grunt watch
+$ grunt
```

Comment:
"npm run build" should take care of that, right now, it looks like you need to run both - grunt and npm commands

Regards,
Oleg

> On 17 нояб. 2015 г., at 19:40, Timmy Willison <notifications@github.com> wrote:
> 
> In CONTRIBUTING.md:
> 
> >  
> >  ```bash
> > -$ grunt && grunt watch
> > +$ grunt
> You do when following those steps to view the test page.
> 
> —
> Reply to this email directly or view it on GitHub.
>